src/main/resources/resources/logisim/strings/std/std_fr.properties

```diff
@@ -813,6 +813,7 @@ TTL7454 = 7454 : Quatre portes ET-OU-OU-INVERTES larges
 TTL74541 = 74541 : Huit tampons / circuits d’excitation
 TTL7458 = 7458 : porte double AND-OR
 TTL7464 = 7464 : 4-2-3-3-2 Porte AND-OR-INVERT
+# ==> TTL74670 =
```

Comment:
banc de registre 4 fois 4 bits avec sorties tri-état

Reply:
Done. Thanks for your french translation.